output/output.go

```diff
@@ -88,3 +92,39 @@ func ZipAndExportOutput(sourcePth, destinationZipPth, envKey string) error {
 
 	return ExportOutputFile(tmpZipFilePth, destinationZipPth, envKey)
 }
+
+// RunAndExportOutput runs a command.Model and directs its ouput to a log file
+// and keeps only the last N lines of the output.
+func RunAndExportOutput(cmd *command.Model, outputPath, envkey string, outputLastLinesOfCode int) error {
+	var outBuffer bytes.Buffer
+	cmd.SetStdout(&outBuffer)
+	cmd.SetStderr(&outBuffer)
```

Comment:
what happens if input `cmd` has stdout or stderr (or both) already set in an outer scope? also if it set then this command should print to the same buffer (at line 120)

cmd := newCommand(...)
cmd.SetStdout(iwantthelast2linesinthisbuffer)
cmd.RunAndExport(cmd, ....)

Reply:
Good question, I do not think there is much we can do about it though. We have to overwrite whatever is set in the outer scope to capture the command's output and this is kind of expected from the function. But at least we could highlight it in the function documentation. 

About your second question, I think we should rather direct line 120 to the originally set Sdtout. WDYT?